tests/metrics/workload_time/cor_create_time.sh.in

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+#  Description of the test:
+#  This test measures the time cc-oci-runtime (cor) takes from creating
+#  a container and before starting it.
+
+set -e
+
+if [ $# -ne 1 ]; then
+	echo -e "Usage: $0 <times to run>" && exit 1
+fi
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../common/test.common"
+
+TEST_NAME="Cor Start Time"
+LOG_FILE="/run/opencontainer/containers/cc-oci-runtime.log"
```

Comment:
This is no longer correct (see commit 35cfef18af7ffd3b79b9462088c798a485ff4602). Here's how the functional tests construct the file - we should really try to make all the common tests code shared):

- https://github.com/01org/cc-oci-runtime/blob/master/tests/functional/common.bash.in#L336